### PR TITLE
Show all errors

### PR DESCRIPTION
**Pull request by spruce**
Add back the ability to show all error from all fields at once.

That ist especially nice after a user tries to submit something to show where something went south.

* Add tests

### CODE REVIEW

**Comment by spruce** [comment]
You think this looks good? @pedrokiefer @GCorbel 

Gonna add some tests soon and want to merge than.

File: addon/components/em-form-group.js [hunk 1]
```diff
@@ -86,6 +88,10 @@ export default Ember.Component.extend(InFormMixin, HasPropertyMixin, HasProperty
       }
     }
   }),
+  canShowErrors: Ember.computed('showAllErrors', 'canShowErrorsFromFocus', function() {
+    return (this.get('showAllErrors')) || this.get('canShowErrorsFromFocus');
```
Comment (GCorbel) [inline addon/components/em-form-group.js]: Extra parenthesis here

**Comment by GCorbel** [comment]
I like it. Very useful.

**Comment by spruce** [comment]
It is already possible by using showErrorsOnRender on the form. So closing this one in favor of that.

**Pull request closed without merging**